tests/test_serialize_sequence.cc

```diff
@@ -564,3 +576,41 @@ TEST_CASE("serialize_sequence: sequence name escaping 2", "[serialize_sequence]"
     // Control char shall be encoded as blank
     REQUIRE(deserialize_seq.get_label().substr(1, 1) == " ");
 }
+
+TEST_CASE("serialize_sequence: : simple step setup", "[serialize_sequence]")
+{
+    Step step_action{Step::type_action};
+    step_action.set_script("a = preface .. 'Bob'");
+
+    Sequence seq{ "test_sequence_with_simple_step_setup" };
+    seq.push_back(step_action);
+    seq.set_step_setup_script("preface = 'Alice calls '");
+
+    serialize_sequence(temp_dir, seq);
+
+    Sequence seq_deserialized = deserialize_sequence(temp_dir + "/test_sequence_with_simple_step_setup");
+
+    REQUIRE(seq_deserialized.get_step_setup_script() == "preface = 'Alice calls '");
+}
+
+TEST_CASE("serialize_sequence: : complex step setup", "[serialize_sequence]")
+{
+    auto step_setup_script =
+R"(
+a = 'Bob'
+        function test(name) return name .. ' with some funky stuff!'     end
+b = test('Alice') \b\b  c = 4)";
+
+    Step step_action{Step::type_action};
+    step_action.set_script("a = preface .. 'Bob'");
+
+    Sequence seq{ "test_sequence_with_complex_step_setup" };
+    seq.push_back(step_action);
+    seq.set_step_setup_script(step_setup_script);
+
+    serialize_sequence(temp_dir, seq);
+
+    Sequence seq_deserialized = deserialize_sequence(temp_dir + "/test_sequence_with_complex_step_setup");
+
+    REQUIRE(seq_deserialized.get_step_setup_script() == step_setup_script);
+}
```

Comment:
Newline at the end of the file is missing.

Reply:
Fixed.